components/mpas-seaice/driver/ice_comp_mct.F

```diff
@@ -2723,6 +2765,57 @@ subroutine freezing_potential(freezingPotential, frazilProduction, seaSurfaceSal
 
    end subroutine freezing_potential!}}}
 
+!***********************************************************************
+!BOP
+!
+! !IROUTINE: datetime
+!
+! !INTERFACE:
+   subroutine datetime(cdate, ctime)
+!
+! !DESCRIPTION:
+! Calculate freezing potential for a cell
```

Comment:
Maybe change this description.

Reply:
Good point. @vanroekel Would you mind quickly changing this? Cheers

Reply:
sure, will do